ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_averages_and_variances.py

```diff
@@ -162,3 +162,48 @@ def test_numpy_std(
         where=where,
         test_values=False,
     )
+
+
+#average
+@handle_frontend_test(
+    fn_tree="numpy.average",
+    dtype_and_a=statistical_dtype_values(function="average"),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.numpy.average"
+    ),
```

Comment:
The current test handlers should not be given the `num_positional_args`. Could you please remove it?